src/page-analysis/content_script/extract-page-metadata.js

```diff
@@ -1,10 +1,18 @@
 import { getMetadata, metadataRules } from 'page-metadata-parser'
+import extractPdfMetaData from 'src/pdf-analysis/extract-pdf-metadata'
 
 // Extract info from all sorts of meta tags (og, twitter, etc.)
 export default function extractPageMetadata({
     doc = document,
     url = window.location.href,
 }={}) {
+
+    if(url.substr(url.lastIndexOf('.')+1) === "pdf" ){
```

Comment:
maybe the substring `.pdf` makes more sense, as we can be sure that URLs that mention "PDF" are note wrongly redirected.